### PR TITLE
Create a new category category-enterprise-query-platform-cn

### PR DESCRIPTION
**Pull request by rootmelo92118**
I create a new category to categorize the enterprise query platform in mainland China.

### CODE REVIEW

File: data/x315 [hunk 1]
```diff
@@ -0,0 +1 @@
+x315.com
```
Comment (IceCodeNew) [inline data/x315:1]: The domain `x315.com` can not be resolved.

File: data/qyyjt [hunk 2]
```diff
@@ -0,0 +1 @@
+qyyjt.cn
```
Comment (IceCodeNew) [inline data/qyyjt:1]: The domain `qyyjt.cn` can not be resolved.

File: data/qinxin [hunk 3]
```diff
@@ -0,0 +1 @@
+qinxin.com
```
Comment (IceCodeNew) [inline data/qinxin:1]: The domain `qinxin.com` can not be resolved.

File: data/hongdunwang [hunk 4]
```diff
@@ -0,0 +1 @@
+hongdunwang.com.cn
```
Comment (IceCodeNew) [inline data/hongdunwang:1]: The site `hongdunwang.com.cn` can not be resolved.

File: data/geolocation-cn [hunk 5]
```diff
@@ -1345,8 +1345,6 @@ zuowen8.com
 zuowenwang.net
 
 # 京ICP备14061319号-2
-jindidata.com
-tianyancha.com
 zhuanxiandai.com
```
Comment (IceCodeNew) [inline data/geolocation-cn:1348]: 这个域名刚才查了下处于可以购买的状态，可以删掉了。

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

File: data/x315 (hunk 1 above)
Comment (rootmelo92118) [reply data/x315:1]: www.x315.com

File: data/qyyjt (hunk 2 above)
Comment (rootmelo92118) [reply data/qyyjt:1]: www.qyyjt.cn

File: data/qinxin (hunk 3 above)
Comment (rootmelo92118) [reply data/qinxin:1]: www.qinxin.com

File: data/hongdunwang (hunk 4 above)
Comment (rootmelo92118) [reply data/hongdunwang:1]: www.hongdunwang.com.cn

File: data/qinxin (hunk 3 above)
Comment (IceCodeNew) [reply data/qinxin:1]: > [www.qinxin.com](http://www.qinxin.com)

Still can not resolve

**Review by IceCodeNew — CHANGES_REQUESTED** [review]
还剩 `qinxin.com` 需要补充信息。

File: data/qinxin (hunk 3 above)
Comment (rootmelo92118) [reply data/qinxin:1]: Sorry I input incorrect domain.
www.qixin.com

**Review by IceCodeNew — APPROVED** [review]

**Pull request merged**